packages/calypso-build/webpack/file-loader.js

```diff
@@ -5,7 +5,7 @@
  * @returns {object}         Webpack loader object
  */
 module.exports.loader = options => ( {
-	test: /\.(?:gif|jpg|jpeg|png|svg)$/i,
+	test: /\.(?:gif|jpg|jpeg|png|svg|mp4)$/i,
```

Comment:
I admit I'm not very familiar with how Webpack loaders work, but should we use a different `outputPath` for `mp4` files if we are using a `videos` folder?

Reply:
I'm not familiar with it either, but I'm not sure why/if it'd be necessary to make any changes due to a different folder based on the fact that no changes were made when the `client/assets/images` path was added by @jsnajdr in #34781. :)

Reply:
I think it's because it doesn't matter where the file is hosted, as long as it matches the extension defined in the loader. Those files are copied to `outputPath` and seems that our webpack config assumes it will be an `images` folder:

https://github.com/Automattic/wp-calypso/blob/452b9a21c280a74dac348c50af2293d6a68079e8/client/webpack.config.js#L114-L131

This means that even if the video is uploaded to the `videos` folder, it will still be served from an `images` path:

<img width="698" alt="Screen Shot 2020-02-28 at 12 12 30" src="https://user-images.githubusercontent.com/1233880/75544353-11ab8f80-5a24-11ea-98c3-3b82c7910362.png">

But maybe that's not a big deal, I don't think there is any specific benefit of having separate folders for images and videos, so maybe we could leave as-is or rename it to something more generic like `files` or `media`. I'll defer to @Automattic/team-calypso on this.